WebApplication1/Services/Validation.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using WebApplication1.BookData;
+
+namespace WebApplication1.Services
+{
+    public class Validation
+    {
+        private bool _valid = true;
+        public List<string> ErrList = new List<string>();
+        public bool isValidBook(Book book)
+        {
+            IdIsValid(book.Id);
+            NameIsValid(book.Name);
+            PriceIsValid(book.Price);
+            AuthorIsValid(book.Author);
+            return _valid;
+          
+        }
+
+        private void AuthorIsValid(string author)
+        {
+            var ans = author.All(Char.IsLetter);
+            if (!ans)
+            {
+                _valid = false;
+                ErrList.Add("The Name of Authors is Invalid, Only Alphabets are allowed");

```

Comment:
all tasks related to ErrList must be done in BookService class to maintain Single Responsibilty